packages/perseus/src/styles/styles.css

```diff
@@ -23,18 +23,6 @@
     padding-top: 32px;
     width: 100%;
 }
-.framework-perseus.perseus-article:not(.perseus-mobile)
-    .perseus-widget-container.widget-float-left
-    .perseus-image-caption
-    .paragraph
-    .paragraph,
-.framework-perseus.perseus-article:not(.perseus-mobile)
-    .perseus-widget-container.widget-float-right
-    .perseus-image-caption
-    .paragraph
-    .paragraph {
-    margin-bottom: 0;
-}
```

Comment:
It didn't look like these were doing anything. Someone please correct me if I'm wrong, but it looked like the styles were coming from `image.css` and the ones here in `styles.css` were obsolete.